src/index.html

```diff
@@ -11,6 +11,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search__bars">
+      <form
+        class="big_search_bar"
+        action="#"
+        method="get"
+        data-qa="big"
+        >
+
+        <label class="input_container">
+          <img
+            src="images/Search.svg"
+            alt="search"
+            class="icon_big"
+            >
+
+          <input
+          class="big-input"
+          type="search"
+          placeholder="Try &#34;Los Angeles&#34;"
+          data-qa="keypress"
+          >
```

Comment:
Code is not aligned, check the checklist
```suggestion
          <input
            class="big-input"
            type="search"
            placeholder="Try &#34;Los Angeles&#34;"
            data-qa="keypress"
          >
```